release.json

```diff
@@ -8,31 +8,15 @@
   "initAppDbVersion": "1.4.0",
   "databaseImageVersion": "1.4.0",
   "agentVersion": "108.0.12.8846-1",
+  "readinessProbeVersion": "1.0.23",
+  "versionUpgradeHookVersion": "1.0.10",
   "openshift": {
     "minimumSupportedVersion": "4.6"
   },
   "search": {
     "version": "0.53.1"
   },
   "supportedImages": {
-    "readinessprobe": {
```

Comment:
Wouldn't it prevent from including those images in RELATED_IMAGES_ in OLM bundles?